helm/flowforge/README.md

```diff
@@ -85,6 +85,7 @@ To use STMP to send email
   - `forge.broker.enabled` (default `false`)
   - `forge.broker.url` URL to access the broker from inside the cluster (default `mqtt://flowforge-broker.[namespace]:1883`)
   - `forge.broker.public_url` URL to access the broker from outside the cluster (default `ws://mqtt.[forge.domain]`, uses `wss://` if `forge.https` is `true`)
+  - `forge.broker.domain` the domain the broker will be hosted on (default `mgtt.[forge.domain]`)
```

Comment:
My only thought here is that we use `forge.domain` to mean a domain we will append hostnames to, where as here we are using `forge.broker.domain` to mean a fully qualified hostname.

Reply:
I see the point. How about renaming to `forge.broker.subdomain` as a subdomain of `forge.domain` ? It will be less confusing and will probably prevent questions about self-hosting MQTT broker separately from the FlowFuse platform.

Reply:
I'm tempted to say `forge.broker.hostname` and label it as needing to be a FQDN (and possibility improve the `forge.entrypoint` doc as that is similar and very vague)